cmd/agent/agent.go

```diff
@@ -0,0 +1,51 @@
+package agent
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+
+	"github.com/SUSE/console-for-sap-applications/cmd/webapp"
+)
+
+var cfgFile string
+
+var cmdRegister = &cobra.Command{
+	Use:   "register [some arguments]",
+	Short: "Register the agent in the system",
+	Long: `Register the agent in the
+system`,
+	Args: cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("You have registered your agent with arguments: " + strings.Join(args, " "))
+	},
+}
+
+var cmdStart = &cobra.Command{
+	Use:   "start [with arguments]",
+	Short: "Start the agent",
+	Long: `Start the
+agent`,
+	Args: cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("You have started the agent with the arguments: " + strings.Join(args, " "))
```

Comment:
we don't need to use different logic from:
https://github.com/SUSE/console-for-sap-applications/blob/main/cmd/webapp/webapp.go#L17-L35

take look there.

Reply:
Now I made it exactly the same.